Makefile

```diff
@@ -338,6 +338,8 @@ test: test-frontend test-backend
 test-backend:
 	@echo "Running go test with $(GOTESTFLAGS) -tags '$(TEST_TAGS)'..."
 	@$(GO) test $(GOTESTFLAGS) -tags='$(TEST_TAGS)' $(GO_PACKAGES)
+# Check if the files in ./build are compilable, they could be ignored because of 'go:build ignore'
+	@find ./build -name '*.go' -exec go build -o /dev/null {} \;
```

Comment:
I am not sure whether it is really needed.

If any one needs the `make fmt` and it breaks in the future, developers can catch the problem in first time.

Otherwise if a tool int `build` is out-dated, I think it can be left there as it is, even if it's broken?

Reply:
I agree with you, but the `gitea-format-imports.go` isn't a build tool used by Makefile, what should we do with it?

Reply:
The file `gitea-format-imports.go` is quite tricky, it only for debug purpose in the first PR https://github.com/go-gitea/gitea/pull/17684 (somewhat I regret that I committed it into the codebase). 

Although it was designed to do some formatting work, but the work is done by `code-batch-process.go` directly. And still now, I can not find any module requires it in the repository since then.

Maybe we can just remove  `gitea-format-imports.go`  to avoid confuses.